js/myparcel.js

```diff
@@ -641,21 +635,20 @@ MyParcel = {
 
 				/* No errors */
 				else {
+                    MyParcel.hideMessage();
 					MyParcel.showPickUpLocations();
                     MyParcel.showDeliveryDates();
-
 					if(MyParcel.data.deliveryOptions.data.delivery.length <= 0 ){
                         MyParcel.hideDeliveryDates();
 					}
-
 					MyParcel.storeDeliveryOptions = response;
-					$('#mypa-deliver-pickup-deliver').click();
 				}
 			})
 			.fail(function(){
 				MyParcel.showFallBackDelivery();
 			})
 			.always(function(){
+				$('#mypa-deliver-pickup-deliver').click();
```

Comment:
`showFallBackDelivery()` (op line 648) doet dit ook. Je hebt dit weggehaald op regel 652. Misschien moet het na regel 645 staan , zodat het in meer gevallen wordt uitgevoegd.

Reply:
Done!